benefit/VERSION.yaml

```diff
@@ -1 +1 @@
-TAG: 5.15.0
+TAG: 5.17.0
```

Comment:
I world suggest keep it 5.15.x rather than 17, currently its fine, you have to update it once your pr after po review

Reply:
updated